resources/views/settings/settings.blade.php

```diff
@@ -0,0 +1,79 @@
+@extends('layouts.main_layout')
+
+@section('head')
+
+    <title>Settings</title>
+
+@endsection
+
+@section('mainBody')
+
+    <main class="primary-main row">
+        <section class="grid register-main">
+
+            <form class="form-horizontal">
+                <fieldset>
+                    <legend>Legend</legend>
+                    <div class="form-group">
+                        <label for="inputEmail" class="col-lg-2 control-label">Nickname</label>
+                        <div class="col-lg-10">
+                            <input class="form-control" id="inputEmail" placeholder="Nickname" type="text">
```

Comment:
The `id` and `for` labels on these are from the login stuff, forgot to change after copy paste?